modules/main_Home/home.jsx

```diff
@@ -3,12 +3,15 @@ import React from 'react';
 import { connect } from 'react-redux'
```

Comment:
if you don't use a /client folder, meteor will restart completely on file change